controllers/solrcloud_controller_test.go

```diff
@@ -176,6 +187,12 @@ var _ = FDescribe("SolrCloud controller - General", func() {
 					},
 					RestartSchedule: "@every 30m",
 				},
+				Availability: solrv1beta1.SolrAvailabilityOptions{
+					PodDisruptionBudget: solrv1beta1.SolrPodDisruptionBudgetOptions{
+						Enabled: true,
```

Comment:
For this one, set `Enabled: false`. That way we can check that it never actually creates one in the first place, and doesn't try to do a delete.

That way you can delete the two lines below:
```
expectPodDisruptionBudget(...)
expectSolrCloudWithChecks(...)
```